src/cephadm/cephadm

```diff
@@ -2641,8 +2646,11 @@ def extract_uid_gid(ctx, img='', file_path='/var/lib/ceph'):
             ).run()
             uid, gid = out.split(' ')
             return int(uid), int(gid)
-        except RuntimeError:
-            pass
+        except RuntimeError as e:
+            ex = (fp, e)
```

Comment:
Do we care about cases of this happening multiple times? For alertmanager we pass a list with two paths for file_path and if both paths caused runtime errors we would overwrite ex.

Reply:
what do you think? Is it worth it? Keep in mind that returning multiple error message makes things much harder to understand and read.